Tests/Unit/Content/ContentTypeResolver/PageSelectionResolverTest.php

```diff
@@ -71,123 +80,106 @@ public function testResolve(): void
+            'content' => [
+                'title' => 'Page Title 2',
+                'excerptTitle' => 'Page Excerpt Title 2',
+            ],
+            'view' => [
+                'title' => [],
+                'excerptTitle' => [],
+            ],
+        ]);
 
-        $result = $this->pageSelectionResolver->resolve($data, $property->reveal(), $locale, ['webspaceKey' => 'sulu']);
+        $result = $this->pageSelectionResolver->resolve(
+            ['page-id-1', 'page-id-2'],
+            $property->reveal(),
+            'en'
+        );
 
         $this->assertInstanceOf(ContentView::class, $result);
         $this->assertSame(
             [
                 [
-                    'id' => '2',
-                    'uuid' => '1',
-                    'nodeType' => 1,
-                    'path' => '/testpage',
-                    'changer' => 1,
-                    'publishedState' => true,
-                    'creator' => 1,
-                    'title' => 'TestPage',
-                    'locale' => 'en',
-                    'webspaceKey' => 'sulu',
-                    'template' => 'headless',
-                    'parent' => '1',
-                    'author' => '2',
-                    'order' => 30,
-                    'description' => 'Main Test Description',
-                    'excerptDescription' => 'Excerpt Test Description',
-                    'excerptCategories' => [
-                        [
-                            'id' => 2,
-                            'key' => 'testcat_1',
-                            'name' => 'TestCat_1_en',
-                            'creator' => 'Adam Ministrator',
-                            'changer' => 'Adam Ministrator',
-                            'medias' => [
-                                [
-                                    'id' => 1,
-                                    'formatUri' => '/media/1/{format}/media-1.jpg?v=1-0',
-                                ],
-                            ],
-                        ],
+                    'id' => 'page-id-1',
+                    'template' => 'default',
+                    'content' => [
+                        'title' => 'Page Title 1',
+                        'excerptTitle' => 'Page Excerpt Title 1',
+                    ],
+                    'view' => [
+                        'title' => [],
+                        'excerptTitle' => [],
+                    ],
+                ],
+                [
+                    'id' => 'page-id-2',
+                    'template' => 'default',
+                    'content' => [
+                        'title' => 'Page Title 2',
+                        'excerptTitle' => 'Page Excerpt Title 2',
+                    ],
+                    'view' => [
+                        'title' => [],
+                        'excerptTitle' => [],
                     ],
                 ],
             ],
             $result->getContent()
```

Comment:
I'm not really happy about the result and having 'view' and 'content' inside `getContent()` method, its very confusing. We should discuss this and if we keep it in getContent this way we should definitely document it when we do things like that else we will get inconsistent very soon in different serializations.

Reply:
dont have a strong opinion about the format of the data. i just want to use the `StructureResolver` for resolving the data to keep things consistent 🙂  for example, i am open to inline the `content` into the root array for example

Reply:
If I rethink about it I think this is the most flexible format we could provide. Its a little bit strange having the `view` in `content` but basically its correct. There are no `view` attributes for the selection content type atleast currently not, in the future there could be something like `present_as` or so it would be false to have directly the pages view array inside there. 

We should just make sure that smart content, articles and entities from the content bundle are serialized into the same way in there selection and smart content providers else it will get confusing.

So all single selections work like this:

```
content.single_selection.content.title
content.single_selection.view.image.displayOption
```

And multi selection and smart contents should return the following way:

```
content.selection[x].content.title
content.selection[x].view.image.displayOption
```

I think we need to document this BC Break, else it will be hard for others to UPGRADE for this which are using the headless bundle already.

Reply:
> We should just make sure that smart content, articles and entities from the content bundle are serialized into the same way in there selection and smart content providers else it will get confusing.

I think we should use the `StructureResolver` for all of them, so they should use the same format automatically. I actually already refactored the resolver for the `PageDataProvider` to use this format in #40 🙂 

> I think we need to document this BC Break, else it will be hard for others to UPGRADE for this which are using the headless bundle already.

Yes, I will add an `UPGRADE.md` file tomorrow. I think we should also make a `0.1` release before merging this?

Reply:
Sounds good all good for me :)